src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs

```diff
@@ -60,6 +61,37 @@ public void CreateDefaultBuilder_RegistersEventSourceLogger()
                 args.Payload.OfType<string>().Any(p => p.Contains("Request starting")));
         }
 
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34580", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
Not sure if this is still needed, but all Hosting tests had this attribute

Reply:
That is to skip running tests on mono. They should be tracked with https://github.com/dotnet/runtime/issues/34582 now that 34580 is closed as its dupe.

Reply:
if the test still fail on Mono, maybe can use issue #34582 instead.

---
In reply to: [443738630](https://github.com/dotnet/runtime/pull/37892#discussion_r443738630) [](ancestors = 443738630)